This failed to build with `JVM crash log found: file:///fineract/hs_err_pid39.log FAILURE: Build failed with an exception.` ... let's rebase and see if it disappears. Otherwise, upgrading JDK sometimes helps for crashes like this.

--- a/fineract-provider/src/main/java/org/apache/fineract/commands/service/AuditReadPlatformServiceImpl.java
+++ b/fineract-provider/src/main/java/org/apache/fineract/commands/service/AuditReadPlatformServiceImpl.java
@@ -75,9 +75,9 @@ import org.springframework.stereotype.Service;
 public class AuditReadPlatformServiceImpl implements AuditReadPlatformService {
 
     private static final Logger LOG = LoggerFactory.getLogger(AuditReadPlatformServiceImpl.class);
-    private static final Set<String> supportedOrderByValues = new HashSet<>(
-            Arrays.asList("id", "actionName", "entityName", "resourceId", "subresourceId", "madeOnDate", "checkedOnDate", "officeName",
-                    "groupName", "clientName", "loanAccountNo", "savingsAccountNo", "clientId", "loanId"));
+    private static final Set<String> supportedOrderByValues = new HashSet<>(Arrays.asList("id", "actionName", "entityName", "resourceId",
+            "subresourceId", "madeOnDate", "checkedOnDate", "officeName", "groupName", "clientName", "loanAccountNo", "savingsAccountNo",
+            "clientId", "loanId", "maker", "checker", "processingResult"));
 
     private final JdbcTemplate jdbcTemplate;
     private final PlatformSecurityContext context;

